.circleci/integration-tests/Dockerfile

```diff
@@ -14,6 +14,18 @@ RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2
 RUN unzip awscliv2.zip
 RUN ./aws/install
```

Comment:
How about combining these two with the RUN command on line 13 using `&&` to avoid additional number of layers? Reference: https://medium.com/devgorilla/docker-best-practices-images-98e9464cc173